apps/graphql/src/apps/common/CommonTypes.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+export type DateType = {|
+  +local: ?(string | number),
```

Comment:
I know it was here before, but I think we shouldn't do it this way. I had problems with types like this one because `format` from new `date-fns` can't accept number and you have to call different function for it. And even without this issue it's still confusing. 

What do you think? I would propose to create new issue for it.

Reply:
The issue is that we get date as string from Tequila, and mocked data for MMB has it as int.

Reply:
@jvaclavik I totally agree and I also tried to update it, but then I find out what Trond mentioned and don't want this PR to grow too big with other date related necessary changes.
But we can definitely create new issue for it.

Reply:
> The issue is that we get date as string from Tequila, and mocked data for MMB has it as int.

I understand the reason, but according to me those types should be separated.

Reply:
Well, they will be passed to the same resolver, so that is why it has this type. 

Maybe we should just wrap both sanitizers with new Date(local)